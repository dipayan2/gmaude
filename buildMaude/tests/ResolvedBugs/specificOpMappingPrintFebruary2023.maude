***
***	Test that specific op maps print correctly.
***

mod FOO is
  sorts Foo Bar .
  op f : Foo -> Bar .
endm

mod BAR is
  inc FOO * (op f : Foo -> Bar to g) .
endm

show desugared .
