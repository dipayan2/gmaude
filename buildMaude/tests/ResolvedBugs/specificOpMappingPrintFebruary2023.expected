mod BAR is
  including BOOL .
  including FOO * (op f : [Foo] -> [Bar] to g) .
endm
Bye.
